core/am/rest-cxf/src/main/java/org/apache/syncope/core/rest/cxf/service/wa/GoogleMfaAuthAccountServiceImpl.java

```diff
@@ -88,6 +98,15 @@ public GoogleMfaAuthAccount findAccountBy(final String key) {
         return result;
     }
 
+    @Override
+    public PagedResult<GoogleMfaAuthAccount> countFor(final String owner) {
+        PagedResult<GoogleMfaAuthAccount> result = new PagedResult<>();
```

Comment:
Same as above.

Reply:
@mmoayyed this seems still to be there...

Reply:
Sorry, it seems we already chose to use this strategy to represent primitive results.